docs/developer-guide/nodejs-api.md

```diff
@@ -19,13 +19,15 @@ While ESLint is designed to be run on the command line, it's possible to use ESL
     * [static getErrorResults()][eslint-geterrorresults]
     * [LintResult type][lintresult]
     * [LintMessage type][lintmessage]
+    * [SuppressedLintMessage type][suppressedlintmessage]
     * [EditInfo type][editinfo]
     * [Formatter type][formatter]
 * [SourceCode](#sourcecode)
     * [splitLines()](#sourcecodesplitlines)
 * [Linter](#linter)
     * [verify()](#linterverify)
     * [verifyAndFix()](#linterverifyandfix)
+    * [getSuppressedMessages()](#getsuppressedmessages)
```

Comment:
This link is broken as there's no `getsuppressedmessages` section in the document.